<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreaterSignUp</class>
 <widget class="QDialog" name="CreaterSignUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>332</x>
     <y>301</y>
     <width>306</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 17px;
background-color: #D9D9D9;
color: (127, 127, 127, 0.83);
font-weight: bold;
font-size: 18px;
font-family: PT Sans;
padding-left: 50px;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="youtubelogo">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>195</y>
     <width>203</width>
     <height>46</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="recoverykey">
   <property name="geometry">
    <rect>
     <x>332</x>
     <y>369</y>
     <width>306</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 17px;
background-color: #D9D9D9;
color: (127, 127, 127, 0.83);
font-weight: bold;
font-size: 18px;
font-family: PT Sans;
padding-left: 50px;</string>
   </property>
   <property name="placeholderText">
    <string>RecoveryKey</string>
   </property>
  </widget>
  <widget class="QLabel" name="profile">
   <property name="geometry">
    <rect>
     <x>343</x>
     <y>13</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>332</x>
     <y>93</y>
     <width>306</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 17px;
background-color: #D9D9D9;
color: (127, 127, 127, 0.83);
font-weight: bold;
font-size: 18px;
font-family: PT Sans;
padding-left: 50px;</string>
   </property>
   <property name="placeholderText">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="age">
   <property name="geometry">
    <rect>
     <x>332</x>
     <y>165</y>
     <width>306</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 17px;
background-color: #D9D9D9;
color: (127, 127, 127, 0.83);
font-weight: bold;
font-size: 18px;
font-family: PT Sans;
padding-left: 50px;</string>
   </property>
   <property name="placeholderText">
    <string>Age</string>
   </property>
  </widget>
  <widget class="QLabel" name="recoverykey1">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>380</y>
     <width>26</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="password1">
   <property name="geometry">
    <rect>
     <x>354</x>
     <y>308</y>
     <width>19</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="age1">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>173</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="email">
   <property name="geometry">
    <rect>
     <x>332</x>
     <y>233</y>
     <width>306</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 17px;
background-color: #D9D9D9;
color: (127, 127, 127, 0.83);
font-weight: bold;
font-size: 18px;
font-family: PT Sans;
padding-left: 50px;</string>
   </property>
   <property name="placeholderText">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLabel" name="name1">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>99</y>
     <width>22</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255, 0);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>402</x>
     <y>21</y>
     <width>184</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red;
font-family: OpenSans;
font-weight: bold;
font-size: 27px;</string>
   </property>
   <property name="text">
    <string>Registration</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>314</width>
     <height>464</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="email1">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>240</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>445</x>
     <y>423</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black;
border-radius: 15px;
color: white;
font-size:16px;
font-weight: bold;
background-color: red;
</string>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
